### PR TITLE
Mes-5619: Office page non-cpc outcome pass

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description

Added DelEx changes to the office page when test output is a Pass.

https://jira.dvsacloud.uk/browse/MES-5619

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2020-09-03 at 07 53 17
Screenshot 2020-09-03 at 07 53 34

### CODE REVIEW

**Review by tomcrawleyy — APPROVED** [review]

File: src/components/common/health-declaration-signed/health-declaration-signed.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  @Input()
+  label: string;
+
+  @Output()
+  healthDeclarationChange = new EventEmitter<boolean>();
+
+  formControl: FormControl;
+  static readonly fieldName: string = 'healthDeclarationCtrl';
+  // constructor(private outcomeBehaviourProvider: OutcomeBehaviourMapProvider) { }
```
Comment (Cullima) [inline src/components/common/health-declaration-signed/health-declaration-signed.ts]: remove if not required

File: src/pages/office/cat-be/office.cat-be.page.html [hunk 2]
```diff
@@ -25,43 +25,117 @@ <h4>Test summary</h4>
 
       <ion-card-content class="card-content-ios" no-bounce no-padding>
         <ion-grid class="grid">
+
```
Comment (Cullima) [inline src/pages/office/cat-be/office.cat-be.page.html:28]: is the app ref required for non delegated tests too?

File: src/pages/office/cat-be/office.cat-be.page.html [hunk 3]
```diff
@@ -144,19 +221,25 @@ <h4>Driving faults</h4>
...
       <button id="reason-for-rekey" class="mes-primary-button padding-x-5 full-width" ion-button
-        (click)="goToReasonForRekey()">
+              (click)="goToReasonForRekey()">
         <h3>Continue</h3>
       </button>
     </ion-col>
+    <ion-col *ngIf="isDelegated">
+      <button id="submit-button" class="mes-primary-button padding-x-5 full-width" ion-button
+            (click)="onSubmit()">
+        <h3>Submit</h3>
```
Comment (Cullima) [inline src/pages/office/cat-be/office.cat-be.page.html:241]: Just checking is 'Submit' the correct working for the button?

File: src/pages/office/office.actions.ts [hunk 4]
```diff
@@ -21,7 +21,6 @@ export class OfficeValidationError implements Action {
   readonly type = OFFICE_VALIDATION_ERROR;
   constructor(public errorMessage: string) {}
 }
-
```
Comment (Cullima) [inline src/pages/office/office.actions.ts]: is this change intended?

**Review by Cullima — APPROVED** [review]
Looks really good. A couple of minor comments / changes

File: src/components/common/health-declaration-signed/health-declaration-signed.ts (hunk 1 above)
Comment (andrewsetterfield) [reply src/components/common/health-declaration-signed/health-declaration-signed.ts]: dam.  theres always one.  will remove

File: src/pages/office/cat-be/office.cat-be.page.html (hunk 2 above)
Comment (andrewsetterfield) [reply src/pages/office/cat-be/office.cat-be.page.html:28]: no its not.  condition added to prevent it (Cat BE, C & D)

File: src/pages/office/office.actions.ts (hunk 4 above)
Comment (andrewsetterfield) [reply src/pages/office/office.actions.ts]: reverted
